### DISCUSSION TITLE
rafi we do edexcel compsci

### DISCUSSION THREAD

**Discussion by smellysox1** [RasperRevision/RasperRevision.github.io#12] [General]
have we got all we need from the edexcel spec

**Comment by RasperRevision** [comment]
we do aqa compsci. Everything is on RasperRevision that is in the aqa specification apart from units 6, 7 and 8, which we haven't covered in class.

**Reply by RasperRevision** [reply]
I believe the info on the school site is outdated, the lesson content and textbooks definitely match the aqa specification

**Comment by smellysox1** [comment]
no we definitely do edexcel. all the powerpoints are edexcel, all the units
are named edexcel

**Reply by RasperRevision** [reply]
This is the edexcel specification: https://qualifications.pearson.com/en/qualifications/edexcel-gcses/computer-science-2020.html

This is the aqa specification: https://filestore.aqa.org.uk/resources/computing/specifications/AQA-8525-SP-2020.PDF

We definitely do AQA - just compare the unit structure. If you want, I can email Mrs Batt to check, but I am sure we do AQA based on the specification.

**Reply by RasperRevision** [reply]
Furthermore, from an announcement on the classroom from November, Mrs Batt writes 'The GCSE book to accompany the course is GCSE (9-1) Computer Science AQA 8525 by S Robson and PM Heathcote, published by PG Online. ISBN: 978-1-910523-22-3'.

Also I don't know which powerpoints you're looking at but mine all say AQA.
